include/deal.II/grid/reference_cell.h

```diff
@@ -450,6 +450,22 @@ class ReferenceCell
    * @{
    */
 
+  /**
+   * Return true if the given point is inside the reference cell of the present
+   * space dimension. This function accepts an additional parameter (which
+   * defaults to zero) which
```

Comment:
Maybe such geometry functions with tolerances should not use default parameters.

I think the documentation does not exactly specify what `eps = 0` means. I would understand the docu as follows: A point lying on the boundary together with `eps=0` will result in a return value of `true`. However, the formulation in the first sentence "if the given point is inside the reference cell ..." goes in a different direction.

I would prefer `tolerance` (often used in `GridTools`) over `eps`. Is it really numerical roundoff in the sense of eps?

Reply:
I made those changes.